### DISCUSSION TITLE
What is semantic HTML?

### DISCUSSION THREAD

**Discussion by olx24** [STRINGLABITSOLUTIONS/Bhojon#222] [Q&A] [answered]
In short.

**Comment by montasim** [comment, marked as answer]
Semantic HTML is a coding style. It is the use of HTML markup to reinforce the semantics or meaning of the content. For example: In semantic HTML <b> </b> tag is not used for bold statement as well as <i> </i> tag is used for italic. Instead of these we use <strong></strong> and <em></em> tags.
